tests/tracer/test_encoders.py

```diff
@@ -638,51 +638,44 @@ def run(self):
     assert unpacked is not None
 
 
-@pytest.mark.parametrize("encoder_cls", ["JSONEncoder", "JSONEncoderV2"])
-def test_json_encoder_traces_bytes(encoder_cls, run_python_code_in_subprocess):
+@pytest.mark.subprocess(params={"encoder_cls": ["JSONEncoder", "JSONEncoderV2"]})
```

Comment:
I was initially confused by the naming of `params`. Could this be instead `parametrize`?